bioblend/galaxy/folders/__init__.py

```diff
@@ -45,7 +45,14 @@ def create_folder(self, parent_folder_id: str, name: str, description: Optional[
             payload["description"] = description
         return self._post(payload=payload, id=parent_folder_id)
 
-    def show_folder(self, folder_id: str, contents: bool = False) -> Dict[str, Any]:
+    def show_folder(
+        self,
+        folder_id: str,
+        contents: bool = False,
+        limit: Optional[int] = 10,
+        offset: Optional[int] = 0,
```

Comment:
Neither of these can be None.

```suggestion
        limit: int = 10,
        offset: int = 0,
```